### DISCUSSION TITLE
github copilot application approval.

### DISCUSSION THREAD

**Discussion by celikcedev** [community/community#128148] [Copilot Conversations] [answered]
My github copilot application was approved on 09.06.224. When will I be able to access its benefits? I would be happy if you could help me.

**Comment by henriquevschroeder** [comment]
When you open a new window in VSCode, if you have the Copilot extension enabled, an alert may appear indicating that your access to Copilot has failed. In this alert, you'll find a button labeled "Sign Up to Copilot" (or something similar). Click this button to be redirected to a page where you can claim free access. I haven't found this page through any other links, so this alert seems to be the only way to access it.

**Reply by celikcedev** [reply]
I used this link to apply for free access: https://education.github.com/discount_requests/application

**Comment by celikcedev** [comment]
I guess I have to wait. The following message appears on the application screen: “Hi, celikcedev! You were last verified as of Jun 9, 2024. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.”

**Reply by henriquevschroeder** [reply, marked as answer]
Yes, you're right! It took about a week for my application to be fully processed and to receive access to all its benefits. You can check the status on the same link you provided. Once your request is approved, you will see a card on the right side turn purple with a message like this:

"Congrats! Your request for @celikcedev was approved and your benefits are now available."
